src/iitkgp_erp_login/read_mail.py

```diff
@@ -16,11 +19,14 @@ def getMailID(service):
     
     return None
 
-def getOTP(OTP_CHECK_INTERVAL):
+def getOTP(OTP_CHECK_INTERVAL,headers: dict[str, str], session: requests.Session, login_details: LoginDetails, log: bool = False):
     creds = generate_token()
     service = build("gmail", "v1", credentials=creds)
     
     latest_mail_id = getMailID(service)
+    request_otp(headers=headers, session=session, login_details=login_details, log=log)
+
+    if log: logging.info(" Waiting for OTP ...")
     while True:
```

Comment:
```suggestion
    request_otp(headers=headers, session=session, login_details=login_details, log=log)
    if log: logging.info(" Waiting for OTP ...")
    
    while True:
```

Just to satisfy my OCD and the way I wrote the code.
Also confirm that you have tested it (obviously you have) but thoroughly tried various cases (if any) which might be affected by this change

Reply:
Yep yep yep